core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -94,6 +96,21 @@ class MessageTest extends JUnitSuite {
     for(v <- messages)
       assertEquals(v.message, m.get(v.message))
   }
+
+  @Test
```

Comment:
I think this can instead be:
```scala
@Test(expected = classOf[UnsupportedVersionException])
def testInvalidMagicByte() {
  // Create a message
  val message = new Message("value".getBytes, "key".getBytes, NoCompressionCodec)
  message.buffer.put(Message.MagicOffset, (Message.CurrentMagicValue + 1).toByte)
  message.buffer.putInt(Message.CrcOffset, (message.computeChecksum() & 0xffffffffL).toInt)
  message.ensureValid()
}
```